frontend/src/features/projects/spl/forms/CloseOutFinancialsForm.tsx

```diff
@@ -26,7 +26,8 @@ const CloseOutFinancialsForm = (props: CloseOutFinancialsFormProps) => {
         <Col md={1}></Col>
         <Col>
           <ProjectNotes
-            field={`notes[${NoteTypes.CloseOut}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption